package.json

```diff
@@ -16,10 +16,7 @@
   "engines": {
     "node": "^9"
   },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/olinlibrary/abe-web-frontend.git"
-  },
+  "repository": "github.:olinlibrary/abe-web",
```

Comment:
This uses npm [shortcut syntax](https://docs.npmjs.com/files/package.json#repository).